### ISSUE TITLE
Add a folder tree

### ISSUE THREAD

**Issue by Surfrat** [aloneguid/StorAMP#7]
Thank you for this awesome software.

I have a group of non techie people managing documents in an Azure blob store.

We used to use Azure Explorer from Cerebrata until they removed it. It's best feature was that it worked in exactly the same way as Windows Explorer and had a tree on the LHS.

Would you please consider adding this feature?

![Azure-Explorer1](https://user-images.githubusercontent.com/2066487/64097301-34563780-cd64-11e9-8b18-47d829f7359a.png)

Thanks

Mark.

**Comment by aloneguid** [comment]
Hi, I might add the folder tree feature in future, sounds like a good idea. I'm not sure about making it look like windows explorer though.

**Comment by Surfrat** [comment]
:)

**Comment by aloneguid** [comment]
quick question - would you pay for a Pro version if this feature is implemented? ;)

**Comment by Surfrat** [comment]
Yes, assuming cost is reasonable.

**Comment by aloneguid** [comment]
It's coming, work on this is in progress 
![image](https://user-images.githubusercontent.com/3155189/69725612-72ba8300-1116-11ea-8ac1-3d0aeace9535.png)

**Comment by Surfrat** [comment]
Very nice...

**Comment by aloneguid** [comment]
What are the most important features for you in that tree? Just a few ideas I'm working on:
- sync selected folder with main view
- support drag'n'drop from windows explorer to a folder
- delete/rename/create folder
- move them around with drag'n'drop

**Comment by TaaheroHiro** [comment]
From @Surfrat  
Most important feature for me is item 1 and 3.
Actually all 4 are really nice features.

**Comment by Surfrat** [comment]
Tested now. Folder tree works well. Could you please allow an option to place it on the left of the list?

Thanks.

**Comment by aloneguid** [comment]
Yes sure, I'm planning to make all the panels dockable and resizeable, it will have to wait until then.
